buyer-api/src/routes/batchDataResponse.js

```diff
@@ -18,16 +19,18 @@ const router = Router();
  *       422:
  *         description: When passphrase is incorrect
  */
-router.post('/', async (req, res) => {
-  if (req.headers.authorization === `Bearer ${config.sendBatchPassphrase}`) {
+router.post(
+  '/',
+  validateAuthorizationToken(config.sendBatchPassphrase),
```

Comment:
What if we load the passphrase in the middleware? So that we just define `validateAuthorizationToken` in the router and then the auth middleware knows how to handle it?

Reply:
We'll work on this to improve the authentication scheme